solr/core/src/java/org/apache/solr/search/SolrDocumentFetcher.java

```diff
@@ -539,6 +541,21 @@ public Reader readerValue() {
     public Number numericValue() {
       return null;
     }
+
+    @Override
+    public StoredValue storedValue() {
+      if (fieldType().stored() == false) {
+        return null;
+      } else {
+        return new StoredValue(stringValue());
+      }
```

Comment:
I guess `fieldType().stored()` will never be `false` here?

Reply:
I think you are correct. will remove the if block